src/sentry/relay/config/metric_extraction.py

```diff
@@ -390,7 +391,25 @@ def _convert_aggregate_and_query_to_metric(
             groupbys=groupbys,
         )
 
+        # TODO: switch to validate_rule_condition
```

Comment:
Is such a function better?
Do we have a task to track the work?
Any impacts of not doing it now?

Reply:
Relay library will expose it in the next version. It should be a drop in replacement since we are just renaming it from `validate_sampling_condition` to `validate_rule_condition` to better reflect its usages